docs/explanations/architecture/styles.md

```diff
@@ -0,0 +1,143 @@
+3. **Access to style data**. A control may need external information about the styles available for a given block: the list of colors, or the list of font sizes, for example. These are called "style presets", as they are a preselection of styles usually defined by the theme, although WordPress provides some defaults. Check the [list of data](https://developer.wordpress.org/block-editor/reference-guides/theme-json-reference/theme-json-living/#settings) a theme can provide to the editor and how a block author can get access to it via [useSetting](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-block-editor/#usesetting).
+4. **Serialize the user style into HTML markup**. Upon an user action, the block HTML markup needs to be updated accordingly (apply the proper class or inline style). This process is called serialization and it is the [edit, save](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-edit-save/), and [render_callback](https://developer.wordpress.org/block-editor/how-to-guides/block-tutorial/creating-dynamic-blocks/) responsibility: these functions take block data and convert it into HTML.
+
+In essence, these are the essential mechanics a block author needs to care about for their block to be able to be styled by the user. While this can be done completely manually, there's an API that automates this process for common style needs: Block Supports.
+
+#### Block Supports API
+
+[Block supports](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-supports/) is an API that allows a block to declare what features it supports. By adding some info to their [block.json file](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/), the block tells the system what kind of actions a user can do to it.
+
+For example:
+
+```json
+{
+  "name": "core/paragraph", 
+  "...": "...",
+  "supports": {
+	"typography": {
+		"fontSize": true
+	}
+  }
+}
+```
+
+In its `block.json`, the paragrah block declares support for font size. It means that there will be an UI control available for users to tweak the font size of that particular block. The system will take care of setting up the UI control data (the font size of the block if it has one already assigned, the list of available font sizes to show), and will serialize the block data into HTML markup upon user changes (attach classes and inline styles appropiately).
+
+By using the block supports mechanism via `block.json`, the block author is able to create the same experience than before just by writing a couple of lines.
+
+Besides the obvious advantage of having to do less work to achieve the same results, there's a few other advantages:
```

Comment:
Tiny nit: because we're including a document here for folks who mightn't already know about the concepts, shall we remove the word `obvious`? Also removed switched to `benefit` to avoid repetition.

```suggestion
Besides the benefit of having to do less work to achieve the same results, there's a few other advantages:
```